packages/common/driver-definitions/api-report/driver-definitions.api.md

```diff
@@ -255,6 +266,28 @@ export interface IResolvedUrl {
     url: string;
 }
 
+// @alpha (undocumented)
```

Comment:
Please be sure to add documentation to all of the new APIs here before merging 🙂

Reply:
yes, this is just draft.